tests/devapps/MSI/HelperService/MSIHelperService/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-ef": {
```

Comment:
What does this json do? Will it install Entity Framework?

Reply:
yes it came as part of the weather sample. removed it